src/builder.spec.ts

```diff
@@ -1397,6 +1397,53 @@ describe('Builder', () => {
             }
         );
     });
+    test('retain target state when equal after normalization', async () => {
+        await runTest(
+            {
+                messagesBefore: '<?xml version="1.0"?>\n' +
+                    '<xliff version="1.2" xmlns="urn:oasis:names:tc:xliff:document:1.2">\n' +
+                    '  <file source-language="de" datatype="plaintext" original="ng2.template">\n' +
+                    '    <body>\n' +
+                    '      <trans-unit id="ID1" datatype="html">\n' +
+                    '        <source>source val&apos; <x equiv-text="{{c}}" id="INTERPOLATION"/></source>\n' +
+                    '      </trans-unit>\n' +
+                    '    </body>\n' +
+                    '  </file>\n' +
+                    '</xliff>',
+                messagesFrBefore: '<?xml version="1.0"?>\n' +
+                    '<xliff version="1.2" xmlns="urn:oasis:names:tc:xliff:document:1.2">\n' +
+                    '  <file source-language="de" target-language="fr-ch" datatype="plaintext" original="ng2.template">\n' +
+                    '    <body>\n' +
+                    '      <trans-unit datatype="html" id="ID1">\n' +
+                    '        <source>source val\' <x id="INTERPOLATION" equiv-text="{{c}}"/>' +
```

Comment:
```suggestion
                    '        <source>source val\' <x id="INTERPOLATION" equiv-text="{{c}}"/>\n' +
```
I think this would correctly test the situation you described in the bug?

Reply:
Yes, you are absolutely right.